west.yml

```
@@ -304,7 +306,8 @@ manifest:
       path: modules/lib/picolibc
       revision: d07c38ff051386f8e09a143ea0a6c1d6d66dd1d8
     - name: segger
-      revision: 5792675a2470d0f3857de1e77bff57b38c28de3b
+      remote: fgrandel
+      revision: fix/rtt-locking
```

No need to add a remote in `west.yml` you can point to the PR's `HEAD`:

```suggestion
      revision: pull/19/head
```

---

@pdgendt Nice. Fixed.